modules/ding_config_import/ding_config_import.info

```diff
@@ -0,0 +1,4 @@
+name = Ding Config Import
+description = Import configuration from YAML file
+core = 7.x
+package = Ding!
```

Comment:
Please remember to include the dependency on `ding_libs`.

Reply:
Added